src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -259,9 +259,12 @@ void mongoInit
   // "If you call multiple ensureIndex() methods with the same index specification at the same time,
```

Comment:
Please add an entry to CHANGES_NEXT_RELEASE about the fix in this PR. Suggestion:

```
- Fix: avoid creating orion database in -multiservice mode (#3179)
```

Reply:
Updated CHANGES_NEXT_RELEASE